### DISCUSSION TITLE
Drafts+

### DISCUSSION THREAD

**Discussion by Sterrestel** [AprilSylph/XKit-Rewritten#777] [Suggestion Box]
Ability to have different tabs/folders for drafts. Ex: tag games, drafted asks, 'to read', etc.

**Comment by marcustyphoon** [comment]
Definitely feature request this to Tumblr support/the wip blog on a Monday, it sounds fun! Probably tough to do in XKit since we can only really format the posts that are currently on the screen and we don't have a great place to store persistent data that's specific to a blog.
